### PR TITLE
Add Tomviz build recipe

### PR DESCRIPTION
**Pull request by cjh1**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/tomviz/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+c_compiler_version:            # [unix]
+  - 7                          # [linux]
+cxx_compiler_version:          # [unix]
+  - 7                          # [linux]
+numpy:
+  - 1.16
+python:
+  - 3.7
+python_impl:
+  - cpython
```
Comment (mrakitin) [inline recipes/tomviz/conda_build_config.yaml:16]: This file is not needed for conda-forge (will use all default values). Please rerender after removing this file.

Comment (cjh1) [reply recipes/tomviz/conda_build_config.yaml]: I have removed the file, it doesn't look like anyone renders the recipe on stage?

Comment (mrakitin) [reply recipes/tomviz/conda_build_config.yaml]: I think you are right, no rerender is needed in this step (at least I don't see I rerendered anything in https://github.com/conda-forge/staged-recipes/pull/10819).

**Comment by cjh1** [comment]
@mrakitin I think we are in pretty good shape here.

**Comment by cjh1** [comment]
@conda-forge/staged-recipes This is ready review, thank you!

**Comment by mrakitin** [comment]
@conda-forge/staged-recipes, for some reason @cjh1's message did seem to ping you. Could you please review?

File: recipes/tomviz/conda_build_config.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+python:
+  - "3.7.* *_cpython"
```
Comment (mrakitin) [inline recipes/tomviz/conda_build_config.yaml:2]: Would be a problem to build for more Python versions?

File: recipes/tomviz/yum_requirements.txt [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+libXcomposite-devel
+libXcursor-devel
+libxcb-devel
+libXau-devel
+libXi-devel
+libXScrnSaver-devel
+libXrandr-devel
+alsa-lib-devel
+xcb-util-devel
+xorg-x11-server-Xvfb
```
Comment (mrakitin) [inline recipes/tomviz/yum_requirements.txt:19]: Should it be duplicated here too? It's currently in `meta.yaml`:
```yaml
     - {{ cdt('xorg-x11-server-xvfb') }}  # [linux]
```
I think it's true for most of the packages in this file. Looks like it's a mixture of system packages with the conda-provided ones.

**Review by mrakitin — COMMENTED** [review]
@cjh1, I have a couple of questions below. Looks good otherwise (I am not a @conda-forge/staged-recipes member, but I was looking at Chris's relevant updates in https://github.com/nsls-ii-forge/tomviz-feedstock/pull/1).

File: recipes/tomviz/conda_build_config.yaml (hunk 2 above)
Comment (cjh1) [reply recipes/tomviz/conda_build_config.yaml:2]: Yes, the build times out.

File: recipes/tomviz/yum_requirements.txt (hunk 3 above)
Comment (cjh1) [reply recipes/tomviz/yum_requirements.txt:19]: Not sure its actually needed, it a carry over from paraview. I am going to try removing it.

Comment (cjh1) [reply recipes/tomviz/yum_requirements.txt:19]: The others need to be in the `yum_requirements.txt`, not sure if they need to be duplicated in `meta.yml` but that seem to be what other recipes do.

**Comment by mrakitin** [comment]
@isuruf, @CJ-Wright, @mariusvniekerk, would you be able to review it? It would be great to have this package for NSLS-II deployment we are planning soon. Thanks a lot!

**Comment by mrakitin** [comment]
Thanks, @ocefpaf!

**Pull request merged**